### DISCUSSION TITLE
【氵贴】新人入驻绳网

### DISCUSSION THREAD

**Discussion by shengjin114514** [share121/inter-knot#1163] [其他]
如题，还有就是阅览固定977，点进去后就是592，大伙都是这样的吗
![7c76abef6f8016cbaa62b467f2fa3a96bbb7b056_raw(1)](https://github.com/user-attachments/assets/d3136b2b-ee84-497c-a6e8-141eb3262610)

**Comment by YukariZZZ** [comment]
作者用的贴图，实际没有显示功能

**Reply by shengjin114514** [reply]
OK，感谢解答
